content/actions/using-workflows/caching-dependencies-to-speed-up-workflows.md

```diff
@@ -322,7 +322,7 @@ Caches have branch scope restrictions in place, which means some caches have lim
 
 For example, a repository could have many new pull requests opened, each with their own caches that are restricted to that branch. These caches could take up the majority of the cache storage for that repository. Once a repository has reached its maximum cache storage, the cache eviction policy will create space by deleting the oldest caches in the repository. In order to prevent cache thrashing when this happens, you can set up workflows to delete caches on a faster cadence than the cache eviction policy will. You can use the [`gh-actions-cache`](https://github.com/actions/gh-actions-cache/) CLI extension to delete caches for specific branches.
 
-This example workflow uses `gh-actions-cache` to delete all the caches created by a branch once a pull request is closed.
+This example workflow uses `gh-actions-cache` to delete up to a 100 caches created by a branch once a pull request is closed.
```

Comment:
Not sure if you prefer `up to a hundred` or `up to a 100`

Reply:
<div style="font-style: normal; font-variant-caps: normal; font-weight: 400; orphans: auto; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-tap-highlight-color: rgba(26, 26, 26, 0.3); -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-family: Roboto, sans-serif; font-size: 15px; letter-spacing: -0.30000001192092896px;">

google-site-verification=4yQFXO3Mw79B_9l_A6i4_BCmO-frjIP1zmFDHpGr_F4 | google-site-verification=4yQFXO3Mw79B_9l_A6i4_BCmO-frjIP1zmFDHpGr_F4 |  
-- | -- | --
google-site-verification=4yQFXO3Mw79B_9l_A6i4_BCmO-frjIP1zmFDHpGr_F4 |  

</div><div class="modal-button-group" style="font-style: normal; font-variant-caps: normal; font-weight: 400; orphans: auto; text-indent: 0px; text-transform: none; white-space: normal; widows: auto; word-spacing: 0px; -webkit-tap-highlight-color: rgba(26, 26, 26, 0.3); -webkit-text-size-adjust: auto; -webkit-text-stroke-width: 0px; text-decoration: none; z-index: 999999; background-color: rgb(234, 234, 234); position: fixed; bottom: 0px; text-align: right; left: -15px; width: 720px; border-top-width: 3px; border-top-style: solid; border-top-color: rgb(191, 191, 191); padding: 10px 15px; caret-color: rgb(0, 0, 0); color: rgb(0, 0, 0); font-family: Roboto, sans-serif; font-size: 15px; letter-spacing: -0.30000001192092896px;"><input type="button" value="Kapat" name="ok" class="btn btn-default hand mr15" style="margin-right: 15px; background-color: rgb(246, 247, 250); color: rgb(51, 51, 51); letter-spacing: normal; display: inline-block; font-size: 16px; font-weight: 400; border-radius: 3px; padding: 6px 12px; border: none; box-shadow: rgba(34, 34, 34, 0.1) 0px -3px 0px 0px inset; -webkit-box-shadow: rgba(34, 34, 34, 0.1) 0px -3px 0px 0px inset; transition: all 0.2s ease 0s; cursor: pointer !important;"></div>

Reply:
```suggestion
This example workflow uses `gh-actions-cache` to delete up to 100 caches created by a branch once a pull request is closed.
```
If the number is over 10 we use numbers instead of spelling it out. I'm just going to delete the "a", then I'll get this merged! Thank you for making this update.